include/lbann/layers/loss/top_k_categorical_accuracy.hpp

```diff
@@ -69,6 +69,11 @@ class top_k_categorical_accuracy_layer : public data_type_layer<TensorDataType>
   std::string get_type() const override { return "top-k accuracy"; }
   data_layout get_data_layout() const override { return T_layout; }
   El::Device get_device_allocation() const override { return Dev; }
+  bool can_run_inplace() const override { return false; }
+  int get_backprop_requirements() const override
+  {
+    return ERROR_SIGNALS | PREV_ACTIVATIONS;
```

Comment:
Maybe I am missing something, but I don't see a backprop method, nor do I see where it requires `PREV_ACTIVATIONS`

Reply:
Added comment.